shell.nix

```diff
@@ -19,13 +19,17 @@ mkShell' {
 
     python3
     python3Packages.pip
+    protobuf
+    protoc-gen-go
+    protoc-gen-go-grpc
+
+    foundry-bin
```

Comment:
These appear to have broken the nix shell for me:
```
error: builder for '/nix/store/g9g5kmnv6hs4bv0hgzf0jv9nnr4k0n3l-source.drv' failed with exit code 1;
       last 7 log lines:
       >
       > trying https://github.com/foundry-rs/foundry/releases/download/nightly-bfc6549f0d50fe31cd2fae875c2c7233db98bde9/foundry_nightly_darwin_amd64.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       >   0     9    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download source from any mirror
       For full logs, run 'nix-store -l /nix/store/g9g5kmnv6hs4bv0hgzf0jv9nnr4k0n3l-source.drv'.
error: 1 dependencies of derivation '/nix/store/yvwz6v83vxqi2razmmdd4lrd0552fsf9-foundry-0.0.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/g4c68z4l24vpgcsgrhqkjx1ki7vb0ii2-nix-shell-env.drv' failed to build
```
https://github.com/foundry-rs/foundry/releases/tag/nightly-bfc6549f0d50fe31cd2fae875c2c7233db98bde9 does not exist. There is no tag for this so it can't download it.